src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -242,29 +253,16 @@ protected void loadItems(Connection conn, int itemKount) {
         batchSize++;
 
         if (batchSize == workConf.getBatchSize()) {
-          itemPrepStmt.executeBatch();
-          itemPrepStmt.clearBatch();
-          transCommit(conn);
+          performInserts(Arrays.asList(itemPrepStmt), conn);
```

Comment:
why do we need to convert this to a list?

Reply:
There are cases where we need to send 3. Prepared statements. Hence made the API accept a list.